package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "firebase-functions",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
Please revert this as well - our automated systems update this when we do a release.

Reply:
Covered in 6bc06f3e27e64f55b0e858aed13f9872381acd3c